source/glossary/web-development.md

```diff
@@ -39,10 +39,10 @@ Both LESS and Sass are types of CSS preprocessor markup intended to give you muc
 [Sass/Less Comparison](https://gist.github.com/chriseppstein/674726)
 
 
-## Linter, linting, jslint, jshint
+## Linter, linting, jslint, jshint, eslint
```

Comment:
I was wondering, maybe we stick to "Linter, linting"  and list the actual linters in the text as examples of linters, and we can even link to the projects?